s3tests_boto3/functional/test_s3.py

```diff
@@ -13430,3 +13430,182 @@ def test_get_object_torrent():
         status, error_code = _get_status_and_error_code(e.response)
         assert status == 404
         assert error_code == 'NoSuchKey'
+
+@pytest.mark.checksum
+def test_object_checksum_sha256():
+    bucket = get_new_bucket()
+    client = get_client()
+
+    key = "myobj"
+    size = 1024
+    body = FakeWriteFile(size, 'A')
```

Comment:
```suggestion
    body = FakeWriteFile(1024, 'A')
```
Since it is not used inside the test except this place, can some some memory to not assign a variable to it :)